rfc/0054-sdk3-management-apis.md

```diff
@@ -1563,6 +1563,7 @@ The SDK should not try to intercept a "bucket.scope.index" name and extract "ind
 The implementation of `ScopeSearchIndexManager` is identical to `SearchIndexManagement`, except for the endpoints and some error handling details:
 
 * `GetIndex`, `UpsertIndex`, `DropIndex`: use endpoint `/api/bucket/{bucketName}/scope/{scopeName}/index/{indexName}` with GET, PUT and DELETE respectively.
+* `AnalyzeDocument`: use endpoint `/api/index/{bucketName}.{scopeName}.{indexName}/analyzeDoc` with POST, there is no scoped endpoint for analyze document.
```

Comment:
Ah, really?  May have a bug in the Java SDK then.

Reply:
Apparently so, from elsewhere (and from someone else)

> They told me:
so there isn’t a scoped endpoint for analyseDoc
you’ll have to provide the fully qualified index name

Reply:
This just seems to be a server bug - why have a scoped index endpoint for every operation except analyzeDoc?